reddwarf/data_loader.py

```diff
@@ -58,6 +60,29 @@ def populate_polis_ids(self):
             # If not set, write it from what's provided.
             self.polis_id = self.report_id or self.conversation_id
 
+    class ReportType(Enum):
+        SUMMARY = "summary"
+        VOTES = "votes"
+        COMMENTS = "comments"
+        PARTICIPANT_VOTES = "participant-votes"
+        COMMENT_GROUPS = "comment-groups"
+
+    def fetch_csv(self, type: ReportType, output_dir=None):
```

Comment:
this is only used in the test (and duplicates calls we already make elsewhere to reportExport endpoints), so think we could move it into the test file itself or maybe `tests/helpers.py`?